### ISSUE TITLE
[nix-repl] Show which variables get loaded

### ISSUE THREAD

**Issue by kstrafe** [NixOS/nix#11404]
It's not practical to run `nix repl nixpkgs` and only get "Added 6 variables". This does not tell me which variables. I have to run the output against a plain `nix repl`, perform tab completion, and then diff these two outputs.

I'd like to just print the variables which were just loaded into the global scope.

**Comment by pbsds** [comment]
should be capped

```
$ nix repl '<nixpkgs>'
warning: future versions of Lix will require using `--file` to load a file
Lix 2.90.0
Type :? for help.
Loading installable ''...
Added 21545 variables.
```

**Comment by kstrafe** [comment]
@pbsds Good point. Perhaps we should add a command for getting the most recently loaded variables and show these in a pager.

**Comment by kstrafe** [comment]
PR https://github.com/NixOS/nix/pull/11406

**Comment by kstrafe** [comment]
Closing as resolved.
